src/app/components/channel-list/channel-list.component.ts

```diff
@@ -59,31 +68,32 @@ export class ChannelListComponent implements OnInit, OnDestroy, AfterViewInit {
         private selectedTokenService: SelectedTokenService,
         private sharedService: SharedService,
         private addressBookService: AddressBookService
-    ) {}
+    ) {
+        this.tokens$ = this.tokenPollingService.tokens$.pipe(
+            map((tokens) =>
+                tokens.sort((a, b) => TokenUtils.compareTokens(a, b))
+            ),
+            shareReplay({ refCount: true, bufferSize: 1 })
```

Comment:
Nice, just keep in mind that refCouting can unsubscribe/resubscribe to the source observable, but usage with `async` pipe is safe as it's done on mount/unmount

Reply:
Thanks, in this case I would not want to keep a subscription if there is no subscriber anymore.